package.json

```diff
@@ -10,7 +10,7 @@
     "webpack": "tsx ./development/webpack/launch.ts",
     "webpack:clearcache": "rimraf node_modules/.cache/webpack",
     "postinstall": "yarn webpack:clearcache",
-    "env:e2e": "SEGMENT_HOST='https://api.segment.io' SEGMENT_WRITE_KEY='FAKE' SENTRY_DSN_DEV=https://fake@sentry.io/0000000 yarn",
+    "env:e2e": "SEGMENT_HOST='https://api.segment.io' SEGMENT_WRITE_KEY='FAKE' yarn",
```

Comment:
@matthewwalsh0 I need it to do real Sentry reporting, is this an okay change to make?  I defined `SENTRY_DSN_DEV` in the CircleCI env vars.  Is there a better way to accomplish this?

Reply:
Would it be safer to set this inside `.circleci/config.yml` on the relevant jobs so devs can still run the tests locally without any Sentry overhead?

Reply:
I wrote that initial message 2 weeks ago and a few things have changed, but I think we have to decide -- do we want to respect what the developer set in their `.metamaskrc` for `SENTRY_DSN_DEV`, or do we want to override it for E2E tests?  If they do not set `SENTRY_DSN_DEV` at all, it will go down the `log('Skipped initialization');` path.

Reply:
Good point.

Maybe the safest compromise is update `getSentryTarget` to use the fake DSN if no `SENTRY_DSN_DEV` is defined.

Meaning all the tests by default still use the fake DSN as before, but now the developer can also use their own as desired by these changes?